catalog/spatial/csw/spatial-csw-transformer/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/converter/GmdConverter.java

```diff
@@ -0,0 +1,247 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package org.codice.ddf.spatial.ogc.csw.catalog.converter;
+
+import java.util.GregorianCalendar;
+import java.util.Locale;
+import java.util.Set;
+
+import javax.xml.datatype.DatatypeConfigurationException;
+import javax.xml.datatype.DatatypeFactory;
+
+import org.apache.commons.lang.StringUtils;
+import org.codice.ddf.spatial.ogc.csw.catalog.common.GmdMetacardType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.thoughtworks.xstream.XStream;
+import com.thoughtworks.xstream.converters.Converter;
+import com.thoughtworks.xstream.converters.MarshallingContext;
+import com.thoughtworks.xstream.converters.UnmarshallingContext;
+import com.thoughtworks.xstream.io.HierarchicalStreamReader;
+import com.thoughtworks.xstream.io.HierarchicalStreamWriter;
+import com.thoughtworks.xstream.io.naming.NoNameCoder;
+import com.thoughtworks.xstream.io.path.Path;
+import com.thoughtworks.xstream.io.path.PathTracker;
+import com.thoughtworks.xstream.io.path.PathTrackingWriter;
+import com.thoughtworks.xstream.io.xml.Xpp3Driver;
+import com.vividsolutions.jts.geom.Envelope;
+import com.vividsolutions.jts.geom.Geometry;
+import com.vividsolutions.jts.io.ParseException;
+import com.vividsolutions.jts.io.WKTReader;
+
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.impl.MetacardImpl;
+
+public class GmdConverter implements Converter {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(GmdConverter.class);
+
+    static final DatatypeFactory XSD_FACTORY;
+
+    static {
+        DatatypeFactory factory = null;
+
+        try {
+            factory = DatatypeFactory.newInstance();
+        } catch (DatatypeConfigurationException e) {
+            LOGGER.error("Failed to create xsdFactory: {}", e);
+        }
+
+        XSD_FACTORY = factory;
+    }
+
+    public GmdConverter() {
+
+        XStream xstream = new XStream(new Xpp3Driver(new NoNameCoder()));
+
+        xstream.setClassLoader(xstream.getClass()
+                .getClassLoader());
+
+        xstream.registerConverter(this);
+        xstream.alias(GmdMetacardType.GMD_LOCAL_NAME, Metacard.class);
```

Comment:
I think we might have some duplicate constants between this PR and @bdeining CswEndpoint updates. It'd be nice to clean those up.